agents/craftassist/tests/test_get_memory.py

```diff
@@ -89,15 +91,15 @@ def test_what_are_you_building(self):
 
     def test_where_are_you_going(self):
         # start moving
-        d = MOVE_COMMANDS["move to 42 65 0"]
+        d = MOVE_COMMANDS["move to 8 65 0"]
```

Comment:
why the agent cannot move to 42 65 0 with destructive set to False? Didn't see any object occupying that space (at least in the code)

Reply:
its outside of the bounds of the test world